weather@mockturtl/README.md

```diff
@@ -81,25 +81,23 @@ Historical and Forecast Weather data service provided by Weatherbit LLC in the U
 
 ### Visual Crossing
 
-Weather service from Visual Crossing Corporation founded in 2003 with headquarters in USA and Germany. [Link](https://www.visualcrossing.com/) 
+Weather service from Visual Crossing Corporation founded in 2003 with headquarters in USA and Germany. [Link Visual Crossing Website](https://www.visualcrossing.com/) 
 
 * Needs an API key, you can [Sign Up here](https://www.visualcrossing.com/weather/weather-data-services#/signup) and grab one
 
 * Provides 1000 Free calls a day
 
-### Climacell
+### Tomorrow.io (ClimacellV4)
 
-Meteorological data from American weather technology company with headquarters in Boston since 2016. [Link](https://www.climacell.co/)
+Meteorological data from American weather technology company with headquarters in Boston since 2016. Changed name from Climacell to Tomorrow.io in March 2021. [Link Tomorrow.io Website](https://www.tomowrrow.io/)
```

Comment:
Please also mention that because of this old ClimacellV4 keys are not working anymore.

The naming change inside the applet is will be included in  https://github.com/linuxmint/cinnamon-spices-applets/pull/3928

Reply:
Done, hope I did it right :)

Reply:
Yes, it's ok, thank you